tools/points-service/api.go

```diff
@@ -23,15 +23,15 @@ func NewPointsAPI(logger *slog.Logger, db *sql.DB, ps *PointsService) *PointsAPI
 }
 
 func (p *PointsAPI) StartAPIServer(ctx context.Context, addr string) error {
-	r := http.NewServeMux()
-	r.HandleFunc("GET /health", p.HealthCheck)
-	r.HandleFunc("GET /last_block", p.GetLastBlock)
-	r.HandleFunc("GET /{receiver_type}/{receiver_address}", p.RecomputePointsForAddress)
-	r.HandleFunc("GET /stats", p.GetTotalPointsStats)
-	r.HandleFunc("GET /all", p.GetAllPoints)
+	r := mux.NewRouter()
```

Comment:
I recommend that you do not make this change on the Gorilla Mux router. We should get rid of it and use the one from the standard library in a separate PR.

Reply:
The standard library doesn't work properly for me. In the effort of getting this into production on time, I'd say keep the gorilla mux and we can move to the standard library one  after we've launched.